pyiron/lammps/base.py

```diff
@@ -1146,7 +1146,8 @@ def _set_selective_dynamics(self):
                             "velocity___constraintz"
                         ] = "set NULL NULL 0.0"
 
-    def _ensure_requested_cell_deformation_allowed(self, pressure):
+    @staticmethod
+    def _ensure_requested_cell_deformation_allowed(structure, pressure, prism=None):
```

Comment:
I don't find this name makes sense for something that returns a structure. (I think I might be at fault here, I know I had my fingers on this corner of the code at some point!) Do you think `_modify_structure_to_allow_requested_deformation` is better? I mean they're both really verbose, but if we're going to be verbose anyhow we might as well be precise :p